template.json

```diff
@@ -1,11 +1,14 @@
 {
   "package": {
     "dependencies": {
-      "@reduxjs/toolkit": "^1.1.0",
-      "@testing-library/jest-dom": "^4.2.4",
-      "@testing-library/react": "^9.3.2",
-      "@testing-library/user-event": "^7.1.2",
-      "react-redux": "^7.1.3"
+      "@reduxjs/toolkit": "^1.4.0",
+      "@testing-library/jest-dom": "^5.10.1",
+      "@testing-library/react": "^10.4.3",
+      "jest-environment-jsdom-sixteen": "^1.0.3",
+      "react-redux": "^7.2.0"
+    },
+    "scripts": {
+      "test": "react-scripts test --env=jest-environment-jsdom-sixteen"
```

Comment:
This will be required till there's a release of CRA with jest 25 - it has been merged to master, but not released.

See `MutationObserver shim removed.` here - https://github.com/testing-library/dom-testing-library/releases/tag/v7.0.0